conf/config.inc.php

```diff
@@ -44,6 +44,8 @@
 $ldap_filter = "(&(objectClass=person)($ldap_login_attribute={login}))";
 $ldap_use_exop_passwd = false;
 $ldap_use_ppolicy_control = false;
+//$secondaries_ldap[0]['ldap_url'] = 'ldap://ldap2.example.com';
+//$secondaries_ldap[1]['ldap_url'] = 'ldap://ldap3.example.com';
```

Comment:
I don't think we should add this in config.inc.php

We can maybe add documentation about this script in doc/